CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Inputs can be validated using operations
   - `validate` parameter in `Input` takes `Operation.instance_name`
 - Logistic Regression with SAG optimizer
+- Minor Changes added doc-string and comments in scratchlgr.
```

Comment:
In this case we don't want to change the `CHANGELOG.md` file, this is because `CHANGELOG.md` describes changes between releases. Since you are modifying something you added which is still unreleased. There is no need to add to the changelog.

```suggestion
```